pkg/rewards/5_goldRfaeStakers.go

```diff
@@ -106,12 +107,23 @@ staker_operator_total_tokens AS (
     FLOOR(staker_proportion * tokens_per_day_decimal) as total_staker_operator_payout
   FROM staker_proportion
 ),
--- Calculate the token breakdown for each (staker, operator) pair
+-- Include the operator_pi_split_snapshots table
+operator_pi_splits_cte AS (
```

Comment:
Nit: we don't use cte in naming anywhere

Reply:
I did that to prevent a conflict with the existing `operator_pi_splits` table in the DB. Needed something to explicitly differentiate it.

Reply:
Not sure why this is needed at all. Why not just reference operator_pi_split_snapshots where you need it? You don’t really gain any advantage by selecting just a few columns from it

Reply:
Ah good point. I don't know why I did it that way. Fixed in [0eb7412](https://github.com/Layr-Labs/sidecar/pull/117/commits/0eb7412b60d8a3a7a25ed2d2316e3b5204aba3fd)